src/main/java/com/sergiofah/integracao/service/ProductService.java

```diff
@@ -0,0 +1,29 @@
+package com.sergiofah.integracao.service;
+
+import com.sergiofah.integracao.model.ProductDTO;
+
+import java.util.List;
+
+public class ProductService {
+
+    public List<ProductDTO> getProductsFromCategoryId(Long id) {
+        Service service = new Service();
```

Comment:
The  `service` attribute mus be a class global object, since class have others methods that use it.